lib/ex_unit/test/ex_unit_test.exs

```diff
@@ -760,15 +762,27 @@ defmodule ExUnitTest do
 
   ##  Helpers
 
-  defp on_exit_reload_config(extra \\ []) do
+  defp on_exit_reload_config() do
```

Comment:
Why are we breaking this in two functions?

Reply:
the original function was passing extra options, but those were set on_exit. But that extra argument was never used (I don't know if it became unused with the latest changes).

The new function uses for the current module and restores the original configuration on exit.
We could stick with the new one, and replace `on_exit_reload_config/2` calls

Reply:
Good catch. Yes, we should have just one function then. :)

Reply:
done